zoe-cli/test/com/adevinta/oss/zoe/cli/testcontainers.kt

```diff
@@ -0,0 +1,57 @@
+package com.adevinta.oss.zoe.cli
+
+import com.adevinta.oss.zoe.core.utils.logger
+import org.testcontainers.containers.GenericContainer
+import org.testcontainers.containers.KafkaContainer
+import org.testcontainers.containers.Network
+import org.testcontainers.containers.output.Slf4jLogConsumer
+
+private const val kafkaVersion = "5.5.1"
+
+/**
+ * Context which holds the started containers for easy access in application setup customizers.
+ */
+object TestcontainersContext {
+    private lateinit var kafkaContainer: KafkaContainer
+    val kafka get() = kafkaContainer
```

Comment:
You could this:
You could do this:
```suggestion
    private lateinit var kafkaContainer: KafkaContainer
          private set
```

Reply:
way better now! Did not think of using a simple private setter on a lateinit var.